site/content/docs/5.0/examples/jumbotron/index.html

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
need to brand align

Reply:
will be adressed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/380